R/groups.R

```diff
@@ -154,24 +154,19 @@ clean_group <- function(id, ask = TRUE) {
     }
 
     recursive_clean_group <- function(id) {
-        # Delete platforms
```

Comment:
I believe I have to delete platfrom before deleting streams.  Need to test here.

Reply:
This file got caught up in the merge inadvertently, it's removed now